ALIGNMENT.md

```diff
@@ -0,0 +1,27 @@
+# Confidential Containers and Other Projects
+
+Confidential Containers are connected to a wide array of projects. Some projects are directly implicated in the design of Confidential Containers. Others might be a good pairing for Confidential Containers, and others might be transparently supported or consumed by or via Confidential Containers. If work needs to be done to align a project with Confidential Containers, it will generally involve reconciling the strict trust model of Confidential Containers and the tradional cloud native trust model.
+
+## Directly Related
+
+Given that Confidential Containers are deployed via containerd/cri-o and Kubernetes, these projects are directly implicated. With Confidential Containers, container images are pulled inside the enclave and are essentially not present on the host. Some changes will be needed higher up the stack to accommodate this. Work is already ongoing in at least one of these communities (containerd) to support Confidential Containers.
+
+## Potentially Related
+
+These projects might pair nicely with Confidential Containers, but are not required.
+
+### SPIFFE/SPIRE
+
+SPIFFE and SPIRE can be used to assign a provable identity to every workload. With some adjustments this might be a good way to deliver secrets to confidential containers or even the containers themselves. The attestation process would most likely have to be linked to platform-specific hardware attestation, perhaps in conjunction with the Attestation Agent. The SPIRE agent would need to be moved inside the enclave and the SPIRE server could not be provided by the CSP. Further investigation is encouraged.
```

Comment:
I don't think we defined CSP, this plus all our attestation discussions does bring us back to some sort of Glossary somewhere perhaps?